doc/03_configuration.md

```diff
@@ -155,11 +155,13 @@ duplicated between `tests.edn` and your project configuration (`deps.edn`,
 However, often people will have these paths in their `deps.edn` anyway (e.g. in
 a `:test` or `:dev` profile), so they are accessible for interactive evaluation
 (REPL, in-buffer eval). In some cases you also do not *want* Kaocha to add
-these, because the way we make sure we have a dynamic classloader so we can add
-entries at runtime can confuse third party tools.
+these, because it can confuse third-party tools. This is because Kaocha needs a
+`DynamicClassLoader` to be able to add classpath entries at runtime. If the
+current classloader is not a `DynamicClassLoader` then we set one up ourselves,
+but there have been cases where this caused issues with other tools.
 
 In these cases you can add `:kaocha.testable/skip-add-classpath? false` to the
```

Comment:
Hi there, I was wondering if we want `:kaocha.testable/skip-add-classpath? true` instead here cause that would actually skip the `add-classpath` statement in the code?

If not apologies, maybe I am not understanding too much the use case in the README...

Reply:
Uh yes I believe you're right

Reply:
Ok no worries I can open a PR if that's the case.